### ISSUE TITLE
Personal Messages sometimes incorrectly fail due to "sending the same message again"

### ISSUE THREAD

**Issue by CaptainMeme** [kestasjk/webDiplomacy#105]
When sending out PMs for the Blind Diplomacy games, I've received the following error message, and the PM has not sent:

"You seem to be sending the same message again, this may happen if you refresh the page after sending a message."

This happened on every PM sent (6 of them, each to a different user). It has happened on two separate occasions. I'm not sure what causes it.

**Comment by CaptainMeme** [comment]
To provide a more full explanation of what has happened each time:

I use the same template for each message, but the contents differ substantially. I write them all up and send them in quick succession, with maybe a 1 second delay between sending each one. However, the error message comes up on every PM, even the first one I try to send.

It's only happened twice out of 40ish times I've adjudicated the phase.

**Comment by TimothyJones** [comment]
When the session data has expired, the code erroneously decides that the message has been sent before.

This is caused because `checkTicket()` relies on the session data. Each form has a 'ticket' that is saved  in the session data when the form is created - this ticket is removed when a form is submitted. If no ticket exists in the session data, the form is rejected. Probably, the check should be the other way round - tickets should be added when a form is submitted instead. Or perhaps this check can be removed once #44 is done.

For now, a workaround can be to refresh the page before sending the PM, but ultimately we should do something about this design.

**Comment by jmo1121109** [comment]
This pm system has been disabled. No longer needed
